tooling/reproducible/ReproducibleBuilds.md

```diff
@@ -63,11 +63,15 @@ before the comparable_patch.sh can be run.
 - Ensure VS2022 SDK is installed and on PATH
 - Compile:
   - cd tooling/src/c
+  - run vcvarsall.bat as your arch needs. Eg: vcvars64.bat on x64 windows
+    - You can set up INCUDES manually but it is not worthy
```

Comment:
Why is it not a good idea?  I think this whole line can be deleted

Reply:
I tried to set it up manually and have not get even clsoe to innclue all probably. On web you can find articles how to set it up. It seesm under cygwin, none was good enough.  I do not have strong opinion on that line. But it seems it is possible to do it manually, bt I was unable to do so.

Reply:
Also this line is windows/cygwin sahred. So I would kept it there.